src/NuGet.Clients/NuGet.CommandLine/Commands/PackCommand.cs

```diff
@@ -79,6 +79,12 @@ public ICollection<string> Exclude
         [Option(typeof(NuGetCommand), "CommandMSBuildPath")]
         public string MSBuildPath { get; set; }
 
+        [Option(typeof(NuGetCommand), "PackageCommandOutputResolvedNuSpec")]
+        public bool OutputResolvedNuSpec { get; set; }
+
+        [Option(typeof(NuGetCommand), "PackageCommandOutputSHA512PackageHash")]
+        public bool OutputSHA512PackageHash { get; set; }
```

Comment:
Should these flags be combined? Or should this be a config option instead? 

I'm worried about having too many knobs on pack, there are already a lot of options.

Reply:
We could combine them. As far as I know NuGet pack does not support the config file option.

Reply:
Pack could take the config file option, I'm not sure that is the best place for it, just an option. 
@rohit21agrawal

Reply:
i'd rather have an option like -InstallPackageToFeedV3 which takes the path to the feed as a value.

Reply:
Will consolidate to one option